README.md

```diff
@@ -147,95 +130,63 @@ After you've created your instance of IConnectionManager, simply call the Connec
 
 ServerInfo and ApiClient will be null if State == Unavailable. Let's look at an example.
 
-
 ``` java
-
-         connectionManager.Connect(new Response<ConnectionResult>(){
-
-            @Override
-            public void onResponse(ConnectionResult result) {
-
-				switch (result.getState())
-				{
-					case ConnectionState.ConnectSignIn:
-						// Connect sign in screen should be presented
-						// Authenticate using LoginToConnect, then call Connect again to start over
-	
-					case ConnectionState.ServerSignIn:
-						// A server was found and the user needs to login.
-						// Display a login screen and authenticate with the server using result.ApiClient
-	
-					case ConnectionState.ServerSelection:
-						// Multiple servers available
-						// Display a selection screen by calling GetAvailableServers
-						// When a server is chosen, call the Connect overload that accept either a ServerInfo object or a String url.
-	
-					case ConnectionState.SignedIn:
-						// A server was found and the user has been signed in using previously saved credentials.
-						// Ready to browse using result.ApiClient
-				}
-
-            }
-
-        });
-
-
+connectionManager.Connect(new Response<ConnectionResult>() {
+    @Override
+    public void onResponse(ConnectionResult result) {
+        switch (result.getState()) {
+            case ConnectionState.ConnectSignIn:
```

Comment:
Shouldnt this just be removed?

Reply:
Probably at some point but I haven't touched the actual behavior of the API yet, just want to get build issues and dependencies sorted out first.